walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -79,15 +94,15 @@
    */
   public Collection<TimeRange> queryHelper(Collection<Event> separateEvents, MeetingRequest request) {
     // Map each {@code Event} to it's {@code TimeRange}.
-    List<TimeRange> TimeRangeList = separateEvents.stream()
-                     .map(a -> a.getWhen())
+    List<TimeRange> timeRangeList = separateEvents.stream()
+                     .map(Event::getWhen)
                      .collect(Collectors.toList());
 
     // Resulting Collection
     Collection<TimeRange> openTimeRanges = new ArrayList();
 
     int currentMinTime = 0;
-    for(TimeRange tr : TimeRangeList) {
+    for(TimeRange tr : timeRangeList) {
         
         // If the end time of the current {@code TimeRange} is less than 
         // {@code currentMinTime} then it should be disregarded 
```

Comment:
This comment applies for lines 117-127, but couldn't select those lines for some reason.

I noticed that we update currentMinTime to tr.end() if tr contains currentMinTime, but also in the else clause if it doesn't. So I'm thinking we could change the current else clause to an else-if clause and remove the current `else if (tr.contains(currentMintTime)`. This might be readable/cleaner to rearrange these conditions if it works. Not entirely sure if this is completely analogous, but worth considering...

```
else if (!tr.contains(currentMinTime)) {
  TimeRange newTr = ...
  if (newTr.duration() >= request.getDuration()) {
    openTimeRanges.add();
  }
}
currentMinTime = tr.end();  
```

Reply:
I actually just changed it to an if-else, since tr will either contain currentMinTime or not.

Reply:
I'm not seeing these changes.

Reply:
In the new commit it's on lines 121 - 131